src/Model/User.php

```diff
@@ -1208,6 +1226,11 @@ public static function create(array $data): array
 
 		DBA::update('user', $fields, ['uid' => $uid]);
 
+		$def_gid_groups = Circle::create($uid, DI::l10n()->t('Groups'));
+		if (!$def_gid_groups) {
```

Comment:
Are you sure about this condition?

Reply:
Your comment is a very polite "this is wrong" :-)

Reply:
It looked wrong to me but I didn't want to assume!